src/.defaultsForValidator.js

```diff
@@ -120,15 +120,14 @@ const defaults = {
       'oas2-api-host': "warning",
       'oas2-api-schemes': "warning",
       'oas2-host-trailing-slash': "warning",
-      'oas2-valid-example': "warning",
-      'oas2-valid-definition-example': "error",
+      'oas2-valid-definition-example': "warning",
       'oas2-anyOf': "warning",
       'oas2-oneOf': "warning",
       'oas3-api-servers': "warning",
       'oas3-examples-value-or-externalValue': "warning",
       'oas3-server-trailing-slash': "warning",
-      'oas3-valid-example': "error",
```

Comment:
It looks like this rule is dropped altogether - is that intentional?

I'm guessing it was intention, just double checking because I'm not as familiar with the Spectral rules

Reply:
Well I removed it from the defaultsForValidator.js because it looks like it was removed from our default spectral ruleset.. @mkistler did you accidentally remove this rule or was there a reason why we didn't want this rule in our ruleset ?

Reply:
I removed this because there isn't a Spectral rule with this name in the current version of Spectral.  I'm guessing that it was there in the past but then broken up into the various `oas-valid-<thing>-example` rules.

I think dropping it here is fine and actually I think we should consider removing support for configuring Spectral rules via `.validaterc` altogether.  We've never documented this, and now that the Spectral configuration is clearly documented, I'm not sure this adds enough value to justify the added complexity and maintenance burden.